.cirrus.yml

```diff
@@ -288,6 +288,9 @@ task:
     - name: Haiku x86_64
       env:
         TARGET: x86_64-unknown-haiku
+    - name: Linux loongarch64 lp64d
+      env:
+        TARGET: loongarch64-unknown-linux-gnuf64
```

Comment:
We're back to using `loongarch64-unknown-linux-gnu` in the final iteration of rust-lang/rust#96971, so the tuple should get restored here.